tests/ha/await_upgrade.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Wait for all nodes which are allowed to upgrade before.
+# Maintainer: Christian Lanig <clanig@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use hacluster;
+use lockapi;
+
+sub run {
+
```

Comment:
Just to comment something :) This blank line is not needed but I don't know if Travis is happy with this or not.